packages/design-system/src/components/WIP/Tabs/variants/TabsKit.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { createContext, forwardRef, ReactElement, Ref, useContext, useMemo } from 'react';
+import { TabInitialState, TabState, useTabState } from 'reakit';
+import TabList, { TabListPropsTypesWithoutState } from '../Primitive/TabList';
+import Tab, { TabPropsTypesWithoutState } from '../Primitive/Tab';
+import TabPanel, { TabPanelPropsTypesWithoutState } from '../Primitive/TabPanel';
+
+export type TabsProps = {
+	children: ReactElement | ReactElement[];
+} & TabInitialState;
+
+const TabsContext = createContext<TabState | null>(null);
+
+const Tabs = ({ children, ...initialState }: TabsProps) => {
+	const tab = useTabState(initialState);
+	const value = useMemo(() => tab, Object.values(tab));
```

Comment:
```suggestion
	const value = useMemo(tab, Object.values(tab));
```
Also, why not `[tab]` in itself as "cache identifier" ?

Reply:
Dunno, that's a carryover from frassinier's implem. I'll dig, thanks!

Reply:
<img width="930" alt="Screenshot 2022-07-21 at 10 49 28" src="https://user-images.githubusercontent.com/1319432/180172319-8e211934-b455-44a0-966b-7a993d5f098a.png">

But it seems like `[tab]` is a valid alternative.